### DISCUSSION TITLE
Followers, Following Spaces

### DISCUSSION THREAD

**Discussion by nzetsu** [community/community#8705] [Other Feature Feedback, Questions, & Ideas]
The remaining space after follower, following does not look nice. Could it be possible to put some space between the two to make them evenly spaced?

**Comment by brianlovin** [comment]
Hey @nzetsu could you share more information, like a screenshot, to make it more clear where the issue is? Thanks!

**Reply by nzetsu** [reply]
Hi Brian, 
thank you for replying. Here is a screenshot. 
![Screenshot_2022-01-05 nzetsu - Overview](https://user-images.githubusercontent.com/64031261/148215354-9b40ad8a-4e14-476e-b505-66f650a98185.png)
It is not an issue but i think i does not look nice like the previous version where the stars where right after the followers following. 
Wish you a nice time in 2022 :)
